code/modules/hydroponics/grown/replicapod.dm

```diff
@@ -42,13 +42,16 @@
 	maturation = 10
 	production = 1
 	yield = 1 //seeds if there isn't a dna inside
+	instability = 15 //allows it to gain reagent genes from nearby plants
 	potency = 30
 	var/volume = 5
+	reagents_add = list(/datum/reagent/water = 0.01) //ensures the resulting podperson will have water blood if no other genes are present
 	var/ckey
 	var/realName
 	var/datum/mind/mind
 	var/blood_gender
 	var/blood_type
+	var/datum/exotic_blood
```

Comment:
why is this a datum?

Reply:
because I saw it as a datum somewhere else, and it worked in this context when I tested it, so I figured id leave it and not risk breaking something completely unrelated by changing it

Reply:
Looking at the code myself, this line literally doesn't do anything and doesn't need to exist.

Reply:
YOSHIMASTER's correct, you don't need this (you don't actually use it)